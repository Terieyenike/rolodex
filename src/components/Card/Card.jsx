import React from 'react';
import style from '../../styles.module.css';

export const Card = ({ name, email, id, company }) => (
  <div className={style.card_box}>
    <img
      src={`https://robohash.org/${id}?set=set2&size=180x180`}
      alt='monster'
    />
    <div className={style.card_details}>
      <h2>{name}</h2>
      <p>{email}</p>
      <p>{company}</p>
    </div>
  </div>
);
