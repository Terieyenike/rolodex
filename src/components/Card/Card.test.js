import React from 'react';
import { shallow } from 'enzyme';
import { Card } from './Card';

describe('Card', () => {
  it('expect to render the Card component', () => {
    expect.assertions(2);
    expect(shallow(<Card />).length).toEqual(1);
    expect(shallow(<Card />)).toMatchSnapshot();
  });
});
