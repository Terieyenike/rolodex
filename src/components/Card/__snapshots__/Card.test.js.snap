// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card expect to render the Card component 1`] = `ShallowWrapper {}`;
